docs/content/docs/deep/summaryTelemetry.md

```diff
@@ -0,0 +1,363 @@
+# Summary Collection
```

Comment:
There should be a section that explains what this document is about and the format of the events.